test_scripts/Polices/App_Permissions/ATF_HP_User_Consent_NO.lua

```diff
@@ -0,0 +1,309 @@
+              code = 0,
+              isSDLAllowed = false},
+            method = "SDL.ActivateApp"})
+        :Do(function(_,data)
+            local RequestId = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+            EXPECT_HMIRESPONSE(RequestId,{result = {code = 0, method = "SDL.GetUserFriendlyMessage"}})
+            :Do(function(_,data)
+                self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = "127.0.0.1"}})
+                EXPECT_HMICALL("BasicCommunication.ActivateApp")
+                :Do(function(_,data)
+                    self.hmiConnection:SendResponse(data.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+                    EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+                  end)
+              end)
+          end)
+      end)
+  end
+
+  function Test:DeactivateApp()
+    self.hmiConnection:SendNotification("BasicCommunication.OnAppDeactivated", {appID = self.applications["SPT"], reason = "GENERAL"})
+    EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "LIMITED"})
+  end
+
+  function Test:UpdatePolicyWithPTU()
+    local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+    EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {{url = "http://policies.telematics.ford.com/api/policies"}}}})
+    :Do(function()
+        self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+          {
+            requestType = "PROPRIETARY",
+            fileName = "filename"
+          }
+        )
+        EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "PROPRIETARY" })
+        :Do(function()
+            local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest",
+              {
+                fileName = "PolicyTableUpdate",
+                requestType = "PROPRIETARY"
+              }, "files/PTU_with_permissions_for_app_1234567.json")
+            local systemRequestId
+            EXPECT_HMICALL("BasicCommunication.SystemRequest")
+            :Do(function(_,data)
+                systemRequestId = data.id
+                self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
+                  {
+                    policyfile = "/tmp/fs/mp/images/ivsu_cache/PolicyTableUpdate"
+                  })
+                local function to_run()
+                  self.hmiConnection:SendResponse(systemRequestId,"BasicCommunication.SystemRequest", "SUCCESS", {})
+                end
+                RUN_AFTER(to_run, 800)
+                self.mobileSession:ExpectResponse(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+              end)
+          end)
+      end)
+  end
+
+end
+Precondition()
+
+--[[ Test ]]
+function Test:Step1_User_Disallow_New_Permissions_After_App_Activation()
```

Comment:
Please check in template https://github.com/CustomSDL/sdl_atf_test_scripts/pull/32  how section Test is organized.
Also add to name of all functions in this section TestStep and remove the numbers.